yjit/src/backend/arm64/mod.rs

```diff
@@ -565,10 +565,10 @@ impl Assembler
                     ldp_post(cb, X29, X30, A64Opnd::new_mem(128, C_SP_REG, 16));
                 },
                 Op::Sub => {
-                    sub(cb, insn.out.into(), insn.opnds[0].into(), insn.opnds[1].into());
+                    subs(cb, insn.out.into(), insn.opnds[0].into(), insn.opnds[1].into());
                 },
                 Op::And => {
-                    and(cb, insn.out.into(), insn.opnds[0].into(), insn.opnds[1].into());
```

Comment:
Actually, we should leave this as a plain `and` because we already lower `Op::Test` to `tst`, which is an alias to `ands`.